src/app/month-calendar/month-calendar.service.spec.ts

```diff
@@ -15,10 +15,11 @@ describe('Service: MonthCalendarService', () => {
   it('should check the generateYear method', inject([MonthCalendarService], (service: MonthCalendarService) => {
     const year = moment('14-01-1987', 'DD-MM-YYYY');
     const selected = moment('14-01-1987', 'DD-MM-YYYY');
-    const genYear = service.generateYear({}, year, [selected]);
+    const genYear = service.generateYear({ monthInRow: 3 }, year, [selected]);
```

Comment:
I don't mind to add it an existing test, but we should also make sure that by default we have the default amount of rows. Another thing, you have added it to a unit test where we also should add an e2e test.